src/ViewClient.js

```diff
@@ -0,0 +1,119 @@
+import ApiConnection from './ApiConnection';
+
+const VIEW_MAX_PAGE_SIZE = 10;
+
+export default class ViewClient {
+    constructor(apiConnection, pageSize = VIEW_MAX_PAGE_SIZE) {
+        if (apiConnection instanceof ApiConnection) {
+            this._apiConnection = apiConnection;
+        } else {
+            this._apiConnection = new ApiConnection(apiConnection);
+        }
+
+        this._maxPageSize = pageSize;
+    }
+
+    /**
+     * Get the data for a view
+     * 
+     * @param {string} viewName - the name of the view to retrieve
+     * @param {Date} startDate - optional starting date to begin retrieving data. inclusive. 
+     * @param {Date} endDate - optional ending date to begin retrieving data. inclusive.
+     * @param {integer} page - page of results to retrieve, defaults to first page = 0
+     * @param {integer} pageSize - how many results per page, defaults to 30
+     * @param {Array} include - optional string array of column names to include in result. Leave empty to return all columns.
+     * @param {function} transformFunc - function to transform results data from the request
+     * @return {Promise<object,any>} The view data
+     * @see https://developers.nexosis.com/docs/services/98847a3fbbe64f73aa959d3cededb3af/operations/5919ef80a730020dd851f232 (TODO : FIX)
+     */
+    get(viewName, startDate = null, endDate = null, page = 0, pageSize = 30, include = [], transformFunc = undefined) {
+        var parameters = {
+            page: page,
+            pageSize: pageSize,
+        };
+        if (startDate) {
+            Object.defineProperty(parameters, 'startDate', {
+                value: startDate,
+                enumerable: true
+            });
+        }
+        if (endDate) {
+            Object.defineProperty(parameters, 'endDate', {
+                value: endDate,
+                enumerable: true
+            });
+        }
+        if (include.length > 0) {
+            Object.defineProperty(parameters, 'include', {
+                value: include.join(','),
```

Comment:
I suspect this is copypasta from the data client, but I don't believe it will work either there or here. We need multiple include parameters passed as `?include=foo&include=bar` vs. `include=foo,bar` for the time being, but tracing this code path through makes it appear that we'll be passing the latter rather than the former. I suggest just setting the array here and making `buildRequest` in `ApiConnection` able to translate array properties into query string parameters.